.github/workflows/main.yml

```diff
@@ -13,12 +13,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '20'
       - name: Install modules
         run: yarn
       - name: Run ESLint
-        run: ./node_modules/.bin/eslint . --ext .js,.jsx,.ts,.tsx
-      - name: Run test
-        run: yarn test
```

Comment:
do you think we can return running linter and test to the workflow?

Reply:
will add test to workflow too